.github/workflows/qns.yml

```diff
@@ -1,16 +1,10 @@
 name: QNS
-
 on:
-  push:
-    branches: ["main"]
-    paths-ignore: ["*.md", "*.png", "*.svg", "LICENSE-*"]
```

Comment:
Say we merge multiple pull requests in a row. After the merges, a QNS test fails. How do we quickly find out which merge introduced the regression?